wasm/checksums.json

```diff
@@ -1,20 +1,74 @@
 {
-    "tx_bond.wasm": "tx_bond.d2faf1ae440d4294bc2bf285c6bc2ffd72b0a7d9515e9635610aba815c8fd97a.wasm",
-    "tx_change_validator_commission.wasm": "tx_change_validator_commission.3dad3a0effb08a3a534afefd87aed88fbb690279689541f4f87663b72bd7cfbf.wasm",
-    "tx_ibc.wasm": "tx_ibc.9ec47393bce02e2f99ddffd1ef2531d1b48b9668c63c3410b0a9a2e8980737f4.wasm",
-    "tx_init_account.wasm": "tx_init_account.8ec265df55e7d724d4eb6aab1563ca3b494ca800c1d5d2fd1b4195bac3857b2b.wasm",
-    "tx_init_proposal.wasm": "tx_init_proposal.43512a0ea022b67627fcd6dd662361d137a251eff85cd92bff417fc01cbe1d74.wasm",
-    "tx_init_validator.wasm": "tx_init_validator.b5b70d24645588b5707a7f7b3ac6e76661d2f8de869b588b240d966526cbd9a3.wasm",
-    "tx_reveal_pk.wasm": "tx_reveal_pk.9acbce90f455e69ecc79474eacebe4ca224da8a6b853d95e9eb85b67a9a1f65a.wasm",
-    "tx_transfer.wasm": "tx_transfer.974cc4123d303619af9d7da742f18c0b84fceef9986f6eebaff72cf3c710b9fb.wasm",
-    "tx_unbond.wasm": "tx_unbond.ca2a09b8b3be8e9c546eec7219fa8890fad6e04e3084922cabc907a164f98af2.wasm",
-    "tx_update_vp.wasm": "tx_update_vp.51d4fcda495567d51379e52f2abd29d54804d2c90066307c1f6c4b61d1b42aec.wasm",
-    "tx_vote_proposal.wasm": "tx_vote_proposal.184eed359ebbd1056db66edfe5b53b11d4ab7e24a24db4f387333984df0443da.wasm",
-    "tx_withdraw.wasm": "tx_withdraw.07fbd5e324fc93370c356db9c6f7eff13fc2886559370ba2dc5c713e10516918.wasm",
-    "vp_implicit.wasm": "vp_implicit.f25c93a729d23562c3a9bd79344c03199981e209e3756fd0cf72da9906b541a4.wasm",
-    "vp_masp.wasm": "vp_masp.b34240e9c3ee0914d68f4669b3ebf5eb55447b0a499954fa6cfcf8a1396df99e.wasm",
-    "vp_testnet_faucet.wasm": "vp_testnet_faucet.497a191e0ef340a723c8b01e6c051a82979a7dcae51842f8ef8e945d5903346c.wasm",
-    "vp_token.wasm": "vp_token.d556e61b8a01b8aae5fb0628ba6a598c97008c19e3cc3d8ca0c514b9c1c85ed9.wasm",
-    "vp_user.wasm": "vp_user.f18423c4e47838a9d2e020196faf37deca6d6ebc5348d595ff3466df81b5879a.wasm",
-    "vp_validator.wasm": "vp_validator.08ab66b0b6bb12a9a56a22db196ff49bc04282d73401902c02c50c63a4fc08cc.wasm"
+    "tx_bond.wasm": {
+        "gas": "160",
+        "hash": "tx_bond.8d0afeab4dc91d9294e3c1e1b5f3cd4f1e13f19e1631b894ccce4a4768d991da.wasm"
```

Comment:
it is cosmetic, but since we're changing the format here, could we make these fields to only contain the hash without the file name?